tests/test_rules.py

```diff
@@ -114,7 +114,7 @@ def test_confirm_correct_mitre_tags(self):
                 for tag in tags:
                     if tag not in self.MITRE_ALL and tag.startswith("attack."):
```

Comment:
```suggestion
                    if tag.startswith("attack.") and tag not in self.MITRE_ALL:
```
should be the faster evaluation